build.xml

```
@@ -416,7 +416,7 @@
             <exclusion groupId="hsqldb" artifactId="hsqldb"/> <!-- outdated version -->
             <exclusion groupId="antlr" artifactId="antlr"/> <!-- outdated version -->
           </dependency>
-          <dependency groupId="net.java.dev.jna" artifactId="jna" version="4.0.0"/>
+          <dependency groupId="net.java.dev.jna" artifactId="jna" version="5.8.0"/>
```

This needs to be at least 5.10.0 to pick up the fix https://github.com/java-native-access/jna/pull/1378

```suggestion
          <dependency groupId="net.java.dev.jna" artifactId="jna" version="5.13.0"/>
```

---

I would support using 5.13, as that's what's used on Cassandra trunk https://github.com/apache/cassandra/blob/830f88d4ab2aaa1f7b88b90747a808d0a4e2a708/.build/parent-pom-template.xml#L548